drizzle-kit/src/serializer/mysqlSchema.ts

```diff
@@ -253,9 +256,8 @@ export const MySqlSquasher = {
 		return { name, columns: columns.split(',') };
 	},
 	squashFK: (fk: ForeignKey) => {
-		return `${fk.name};${fk.tableFrom};${fk.columnsFrom.join(',')};${fk.tableTo};${fk.columnsTo.join(',')};${
-			fk.onUpdate ?? ''
-		};${fk.onDelete ?? ''}`;
+		return `${fk.name};${fk.tableFrom};${fk.columnsFrom.join(',')};${fk.tableTo};${fk.columnsTo.join(',')};${fk.onUpdate ?? ''
+			};${fk.onDelete ?? ''}`;
```

Comment:
Was this intended to be changed? It doesn't look different.

Reply:
It's formatted by local prettier and I revert this unnecessary change.